### PR TITLE
Student Info Added

### PR DESCRIPTION
**Pull request by kencyk99**
Added New Info

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://kencyk99.github.io/gci15.fossasia.org

File: _data/students.yml [hunk 1]
```diff
@@ -289,4 +289,11 @@
   image: yeezhi.png
   facebook: yeezhi.leow
   twitter: yee_zhi_leow
-  bio: Eat_Sleep_Ball
\ No newline at end of file
+  bio: Eat_Sleep_Ball
+  
+  -name: Kenneth Chua
```
Comment (jigyasa-grover) [inline _data/students.yml]: There should a space between `-` and `name` , I guess this is the reason the build is failing.

**Comment by kencyk99** [comment]
yes fixed it

**Comment by jigyasa-grover** [comment]
@kencyk99 Cool. Merged :)

**Pull request merged**